flink/v2.0/flink/src/main/java/org/apache/iceberg/flink/sink/dynamic/TableMetadataCache.java

```diff
@@ -80,6 +87,14 @@ ResolvedSchemaInfo schema(TableIdentifier identifier, Schema input) {
     return schema(identifier, input, true);
   }
 
+  ResolvedSchemaInfo getResolvedSchemaInfo(TableIdentifier identifier, Schema input) {
+    CacheItem tableCacheItem = tableCache.get(identifier);
+    if (tableCacheItem == null) {
+      return null;
+    }
+    return tableCacheItem.inputSchemas.get(input);
+  }
+
```

Comment:
Why is this method?
Is it for testing only?
Could we just reuse an existing one?